modules/island-cluster/main.tf

```diff
@@ -0,0 +1,219 @@
+resource "google_service_account" "gke-sa" {
```

Comment:
After going through the module, I believe this would make more sense as an example rather than a module. I would recommend moving this config to the example (rather than the example invoking this modules).

Reply:
done, converted the submodule to an example.